test/FunctionalTests/SimpleAppTest_CoreCLR.cs

```diff
@@ -38,5 +39,32 @@ public class SimpleAppTest_CoreCLR :
                 TestEmbeddedResource.AssertContent("SimpleAppTest.Home.Index.txt", response);
             }
         }
+
+        [Fact]
+        public async Task Precompilation_PreventsRefAssembliesFromBeingPublished()
+        {
+            using (StartLog(out var loggerFactory))
+            {
+                // Arrange
+                var deployment = await Fixture.CreateDeploymentAsync(loggerFactory);
+
+                // Act & Assert
+                Assert.False(Directory.Exists(Path.Combine(deployment.ContentRoot, "refs")));
+            }
+        }
+
+        [Fact]
+        public async Task Precompilation_PublishesPdbsToOutputDirectory()
+        {
+            using (StartLog(out var loggerFactory))
+            {
+                // Arrange
+                var deployment = await Fixture.CreateDeploymentAsync(loggerFactory);
+                var pdbPath = Path.Combine(deployment.ContentRoot, Fixture.ApplicationName + ".PrecompiledViews.pdb");
+
+                // Act & Assert
+                Assert.True(File.Exists(pdbPath), $"PDB at {pdbPath} was not found.");
```

Comment:
The string is printed when the assertion fails. In both of these cases, the pdb could not be found.